benchmarks/maxstar_model_configs.py

```diff
@@ -0,0 +1,459 @@
+"""Shared Benchmark config for v6e orchestrations."""
```

Comment:
Can we have a more descriptive comment at the top.

If this file is meant to be v6e specific it should be renamed to maxtext_v6e_configs or similar. If not v6e specific we should change the comment to reflect this

Reply:
Generally I would prefer maxtext over maxstar, we can always change the name later if at some point add non-maxtext configs

Reply:
I think this is v6e specific tuned model configs. I will rename it ``` maxtext_v6e_model_configs ```. What do you think? @Obliviour